### PR TITLE
(no-ticket) Toolbox feature - inquirer only for GUI 

### PR DESCRIPTION
**Pull request by tudny**
inquirer is not required when using `./easy_toolbox.py <arg>` 
The package is imported only when using ~~CLI~~ GUI

### CODE REVIEW

**Comment by tudny** [comment]
Also, added `.fleet` to `.gitignore` as someone may use it. Same as adding `.vscode` some time ago.

File: easy_toolbox.py [hunk 1]
```diff
@@ -110,6 +109,7 @@ def get_action_from_args() -> Option:
 
 
 def get_action_from_cli() -> Option:
```
Comment (twalen) [inline easy_toolbox.py:111]: actually this is a good moment to fix the function name: `get_action_from_cli` -> `get_action_from_gui`

File: easy_toolbox.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # pip requirements:
 #   python ^3.6
-#   inquirer
+#   inquirer     (only for CLI)
```
Comment (twalen) [inline easy_toolbox.py:5]: CLI (Command Line Interface) -> GUI

Comment (tudny) [reply easy_toolbox.py]: done

File: easy_toolbox.py (hunk 1 above)
Comment (tudny) [reply easy_toolbox.py]: done

**Review by twalen — APPROVED** [review]

**Pull request merged**